DeAceroDLabs/marketplace-frontend
# Feature/dmp 417

## What is done

[[DMP-417] Loading state cards](https://deacero.atlassian.net/browse/DMP-417)

## How is done

- Added prop of loading to `Card` component that will be used on home and search page for the display of cards while loading.
- On Home page `catalogo de productos` there will be 6 loading cards.
- On Search page  there will be 12 loading cards.

## How to test

1. Run the project via `npm start`
2. You should see the Home Screen. There you will see how the cards are loading .
3. You can search for `Clavo` in the search bar to see the search page loading cards.
